docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -21,168 +21,109 @@ helpviewer_keywords:
 
 Naming rules concern the naming of .NET programming language code elements, such as classes, properties, and methods. For example, you can specify that public members must be capitalized or that private fields must begin with `_`.
 
-For each naming rule, you must specify the symbols it applies to, a naming style, and a severity for enforcing the convention, using the properties that follow. The order of the properties is not important.
+A naming rule has three parts:
 
-To begin, choose a title for your naming rule that you will use in each of the properties that are needed to fully describe the rule. For example, `public_members_must_be_capitalized` is a good, descriptive name for a naming rule. This page will refer to the title you choose as **<namingRuleTitle\>** in the sections that follow.
+* the symbol group -- to which symbols does the rule apply?
+* what naming style to associate with the rule, and
+* the severity for enforcing the convention.
 
-## <a name="rule-id-ide1006-naming-rule-violation"></a>Rule ID: "IDE1006" (Naming rule violation)
+## General syntax
 
-All naming options have rule ID `IDE1006` and title `Naming rule violation`. Severity of naming violations can be configured in an EditorConfig file with the following syntax:
+To define a naming rule, symbol group, or naming style, write one or more property settings with the following syntax:
```

Comment:
```suggestion
To define a naming rule, symbol group, or naming style, add one or more property settings using the following syntax:
```

Reply:
I think _add_ implies that some part of the definition exists somewhere else, which isn't true -- once you have a property setting, you have a definition.

Reply:
Then I'd suggest "...define one or more...". "Write" is not a term we generally use in the docs to mean that.

Reply:
Would "... set one or more properties using the following syntax:" be better?

Reply:
That sounds good!